### DISCUSSION TITLE
Light AND dark pygments styles are applied

### DISCUSSION THREAD

**Discussion by lornajane** [pradyunsg/furo#143] [Potential Issue] [answered]
I am loving Furo and the dark mode features, but I've run into a problem with code blocks. (screenshot attached).

What happens is that the light pygments theme is also applied in dark mode, then the dark pygments theme is also applied. This causes a problem where the light theme sets a background colour but the dark theme does not. If I set both light and dark themes to the same dark theme, everything is great in dark mode!

I think that for the pygments themes specifically, only one stylesheet or the other should be loaded, rather than one and then an override which works well for "normal" things. I couldn't immediately see how to fix this ... any ideas?
![Screenshot_2021-05-08 API Examples - Aiven Developer documentation](https://user-images.githubusercontent.com/172607/117536467-12aa5a80-aff3-11eb-82a4-6f94d2d55565.png)

**Comment by pradyunsg** [comment, marked as answer]
Hi! Could you please go ahead and file an issue with steps to reproduce this?

https://github.com/pradyunsg/furo/issues/new?assignees=&labels=&template=issue.yml
